Directory.Build.props

```diff
@@ -1,13 +1,18 @@
 <Project>
     <!-- Set default properties for all projects (can be overridden per project) -->
     <PropertyGroup>
+        <!-- Fix for dotnet build CLI -->
+        <!-- MSBuildThisFileDirectory refers to the folder where Directory.Build.props is defined -->
+        <NitroxRootDir>$(MSBuildThisFileDirectory)</NitroxRootDir>
+        <!-- What version of .NET Core to use for building certain projects -->
+        <NitroxNetCoreVersion>net6.0</NitroxNetCoreVersion>
```

Comment:
`NitroxNetCoreVersion` can be removed. (Minor but `.NET` is the new name for `.NET core`).